misc/systemd/kubo.service

```diff
@@ -1,19 +1,19 @@
 # This file will be overwritten on package upgrades, avoid customizations here.
 #
 # To make persistent changes, create file in 
-# "/etc/systemd/system/ipfs.service.d/overwrite.conf" with 
-# `systemctl edit ipfs.service`. This file will be parsed after this 
+# "/etc/systemd/system/kubo.service.d/overwrite.conf" with
+# `systemctl edit kubo.service`. This file will be parsed after this
 # file has been parsed.
 #
 # To overwrite a variable, like ExecStart you have to specify it once
 # blank and a second time with a new value, like:
 # ExecStart=
-# ExecStart=/usr/bin/ipfs daemon --flag1 --flag2
+# ExecStart=/usr/bin/kubo daemon --flag1 --flag2
```

Comment:
This needs to be restored to `ipfs` – we did not rename the binary.

Reply:
But that's planned, right?

I planned to link the old name to the binary in the new location, as installing "kubo" but getting an "ipfs" binary makes not a lot of sense to me, but on the other hand having backward compability is usually a good thing (for a while).

Reply:
I decided to keep the binary IPFS but make a symbolic link from kubo to ipfs.

This way the kubo package has a "binary" which can be accessed via `kubo param1 param2`

It also allows to keep the binary name of `kubo` in the `kubo.service` file. I think I would break conventions when installing a `ipfs.service` service file with accesses an `ipfs` binary but it's installed via the package name `kubo`.